scenes/src/app/scenes/form-field/form-field-scene.html

```diff
@@ -0,0 +1,6 @@
+<mat-form-field appearance="outline">
```

Comment:
We previously decided that the `fill` appearance would be what we used as the default in the docs. We should probably use that here and have the screen shot reflect that.